src/ocamlorg_frontend/pages/problems.eml

```diff
@@ -48,12 +53,22 @@ Learn_layout.render
         problems is available on
         <a href="https://github.com/ocaml/ocaml.org/tree/main/data/problems">GitHub</a>.
       </p>
+      <p>
+        Every exercise has a difficulty level, ranging from beginner to advanced. 
+        This difficulty level is indicated with the symbol "☡".
+        The absence of marks represents the beginner level, one mark the intermediate level and two marks the advanced level.
+      </p>
     </div>
     <% problems |> List.iter (fun (problem : Ood.Problem.t) -> %>
     <div x-data="{statement: true}" id="<%s" problem.number %>
       >
       <div class="flex space-y-4 lg:space-y-0 lg:items-center flex-col lg:flex-row lg:justify-between">
```

Comment:
```suggestion
      <div class="flex space-y-4 lg:space-y-0 flex-col lg:flex-row lg:justify-between">
```

|before|after|
|-|-|
|![Screenshot 2023-03-15 at 12-27-23 Exercises](https://user-images.githubusercontent.com/6594573/225297757-ca554bcf-c8d3-418d-9675-8abafe741672.png)|![Screenshot 2023-03-15 at 12-27-38 Exercises](https://user-images.githubusercontent.com/6594573/225297774-3c3041ef-8fd8-44d9-9732-d0fb46b36a82.png) align difficulty symbol at the top|